app/views/katello/api/v2/repositories/base.json.rabl

```diff
@@ -2,7 +2,7 @@ object @resource
 
 extends 'katello/api/v2/common/identifier'
 
-attributes :pulp_id => :backend_identifier
+attributes :pulp_id => :pulp_id
```

Comment:
does a change like this require a Katello 4.0 release?

Reply:
whoops that was unintended, i will fix.

Reply:
aw man, i was hoping to get a 4.0 out of this PR :smile_cat: